theme/static/css/hydroshare_core.css

```diff
@@ -1473,12 +1473,16 @@ body > .container{
 
 #citation-text {
     display: inline-block;
-    float: left;
-	background:#EEE;
+    width: 100%;
+    min-height: 30px;
+    background: #EEE;
+    border: 1px solid #cccccc;
+    border-radius: 0px;
+    padding: 4px;
 }
 
 #citation-text a, .rights-url a{
-    word-break: break-word;
+    word-break:normal;
```

Comment:
Got it. Will revert back to break-word style and push it after I rebase with latest develop branch that has core metadata refactoring just merged which triggered some conflicts with my branch I need to resolve.